addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -173,7 +173,6 @@ $o-we-toolbar-height: 32px;
 }
 
 @mixin o-w-preserve-modals {
-    background: 0;
```

Comment:
Ok you found it here, still to change, when moving a bug you can fix it instead ;)